### ISSUE TITLE
It would be helpful to have tooltips on most of the buttons in the UI. One example is the "Share" button above the results table.

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/Feedback#243]

**Comment by Genomewide** [comment]
We need to check the accessibility requirements for this.

**Comment by dnsmith124** [comment]
In the meantime I've updated the share icon to something a bit more common.

**Comment by sierra-moxon** [comment]
from TAQA:
David: accessibility requirements make expanding the tooltips not the right solution here.  UI needs to do more research.

**Comment by gprice1129** [comment]
If tooltips aren't the right solution can we close this ticket @sierra-moxon ?

**Comment by sierra-moxon** [comment]
:) - good question! I think my note from TAQA was me recording what David said in the meeting about this ticket. I have no decision-making power on this ticket, tooltips have always been what have been used in other projects (with admittedly variable access to what makes something accessible for all users).

**Comment by dnsmith124** [comment]
I've done a fair bit more research on this since I spoke about that in TAQA @sierra-moxon, and I think I may have been incorrect with regard to tooltips being inherently inaccessible. Provided that we're very diligent with our aria labeling and keyboard navigation, and ensure that tooltips do not obscure other content I think adding more of them should be fine. 

I'm working on a list of new tooltips and will update this ticket with them later today once that list is complete.

**Comment by desiyopp** [comment]
Yes! It would be okay for things like unlabeled icons, and making sure they aren't blocking any content. The tooltips should also have the arrows pointing to which icon it applies to. 
Also the tooltips should be visible when hovered over with the mouse, _and_ keyboard focusable.

**Comment by dnsmith124** [comment]
After a thorough review I'm going to start by adding tooltips to the share icon and the result download icon.

**Comment by dnsmith124** [comment]
closing this in favor of #525, will track further work there.
